src/odfesqlodbc/es_statement.cpp

```diff
@@ -148,6 +151,36 @@ RETCODE ExecuteStatement(StatementClass *stmt, BOOL commit) {
     return CleanUp();
 }
 
+SQLRETURN GetNextResultSet(StatementClass *stmt) {
+    if (stmt == NULL) {
+        return SQL_ERROR;
+    }
+    ConnectionClass *conn = SC_get_conn(stmt);
```

Comment:
Can we add some newlines for readability into the code? Generally after the completion of a block you would have a newline.

Reply:
added